scavenger-api/src/main/kotlin/com/navercorp/scavenger/dto/ExportMethodInvokationDto.kt

```diff
@@ -0,0 +1,29 @@
+package com.navercorp.scavenger.dto
+
+import com.navercorp.scavenger.entity.ExportMethodInvocationEntity
+
+data class ExportMethodInvokationDto(
+    val id: Long,
+    val modifiers: String,
+    val signature: String,
+    val createdAt: String,
+    val lastSeenAtMillis: String,
+    val invokedAtMillis: String,
```

Comment:
Instead of Millis.. It might be better to use same Instant type for lastSeenAtMillis and invokedAtMillis.
lastSeenAt / invokedAt

Reply:
I have a question and would like to ask you for a more specific answer. Did I understand your desired modification direction well?

```
data class ExportSnapshotMethodDto(
    // ...
    val lastInvokedAt: Instant
) {
    companion object {
        fun from(entity: ExportSnapshotMethodEntity): ExportSnapshotMethodDto {
            return ExportSnapshotMethodDto(
                // ...
                lastInvokedAtMillis = Instant.ofEpochMilli(entity.lastInvokedAtMillis ?: 0)
            )
        }
    }
}
```

Because I modified it based on other DTO classes like SnapshotDto or MethodInvocationDto, they used the Long type for the time variables. That is why I was a little confused..

Reply:
The reason why we used long time for those fields is that it comes from legacy (codekvast)
However it's not for expose to user directly, so we just used it internally.

The your PR make those field(long type) explictly exposed to users by csv format. It's not the expected way.
For scavenger users, I think it's better to convert it to instant format to keep consistency b/w various fields.